libsplinter/src/biome/key_management/mod.rs

```diff
@@ -21,7 +21,7 @@ pub(in crate::biome) mod store;
 use database::postgres::models::KeyModel;
 
 // Represents a public and private key pair
-#[derive(Serialize)]
+#[derive(Serialize, Clone)]
```

Comment:
This is the internal (native, in other parlance) object for Key.  The key-management rest api should really use a separate serialized view, so we can have a clean separation between what is and isn't exposed over the wire.

Reply:
Also created an object, `ResponseKey` for this object. And converted another endpoint which had been returning this internal Key object to the new ResponseKey